teps/0092-scheduling-timeout.md

```diff
@@ -114,7 +117,7 @@ Other options:
 
 ## Proposal
 
-We add configuration to both `PipelineRuns` and `TaskRuns` to allow configuring this new timeout for `TaskRuns`. When
+ add configuration to both `PipelineRuns` and `TaskRuns` to allow configuring this new timeout for `TaskRuns`. When
 specified, this timeout would apply before any other timeouts are started.
```

Comment:
This sentence is no longer true; we're adding two new timeouts, and it's a bit confusing to say they apply before any other timeouts are started.